lib/widgets/witnet/transactions/value_transfer/create_dialog_box/vtt_builder/01_recipient_step.dart

```diff
@@ -532,7 +530,7 @@ class RecipientStepState extends State<RecipientStep>
     return [
       Text(
         localization.withdrawalAddress,
-        style: theme.textTheme.titleSmall,
+        style: theme.textTheme.titleMedium,
```

Comment:
The label should be white in dark mode
![Screenshot 2024-10-07 at 16 36 14](https://github.com/user-attachments/assets/a954f140-ea96-4a65-843b-4516d6585db4)